services/user.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        **other_fields) -> None:

```

Comment:
```suggestion
        **kwargs) -> None:
```
It's better to follow the common way of naming